README.md

```diff
@@ -114,61 +121,144 @@ The milestone needs testing on all platforms. Please report issues when found.
         --add Microsoft.VisualStudio.Component.VC.ATLMFC
     ```
 
-2. Open a “x64 Native Tools Command Prompt” and run:
+    **NOTE:**
+    Builds _must_ be performed in the “x64 Native Tools Command Prompt” (or PowerShell) provided by Visual Studio, _not_ a standard Command Prompt (or PowerShell). The Native Tools CLIs will already have vcpkg installed and configured for use, so no dependency management is needed.
```

Comment:
There is no “x64 Native Tools PowerShell”. I would remove that first reference to PowerShell:

```suggestion
    Builds _must_ be performed in the “x64 Native Tools Command Prompt” provided by Visual Studio, _not_ a standard Command Prompt (or PowerShell). The Native Tools CLIs will already have vcpkg installed and configured for use, so no dependency management is needed.
```